<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CControlRobot</class>
 <widget class="QMainWindow" name="CControlRobot">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>301</width>
    <height>328</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Control Robot</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QSlider" name="horizontalSliderGirar">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="maximum">
     <number>99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonPatear">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>110</y>
      <width>101</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Patear</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonCoger">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>170</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Coger</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSoltar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>200</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Soltar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonAlmacenar">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>230</y>
      <width>101</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Almacenar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonParar">
    <property name="geometry">
     <rect>
      <x>34</x>
      <y>212</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Parar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditIdRobot">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
   </widget>
   <widget class="QSlider" name="verticalSliderAvanzar">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>109</y>
      <width>22</width>
      <height>91</height>
     </rect>
    </property>
    <property name="minimum">
     <number>-99</number>
    </property>
    <property name="value">
     <number>0</number>
    </property>
    <property name="orientation">
     <enum>Qt::Vertical</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonSeleccionarRobot">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>70</y>
      <width>111</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Seleccionar Robot</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxEmpujar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>100</y>
      <width>91</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Empujar</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditServidor">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>10</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>127.0.0.1</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="checkBoxRemoto">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>70</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Remoto</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>301</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
